src/components/molecules/__tests__/Filters.tsx

```diff
@@ -0,0 +1,101 @@
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import React from 'react';
+import { createTransactionCtx, TransactionCtx } from '../../../contexts/transaction';
+import { ITransaction, TransactionType } from '../../../declarations/transaction';
+import { createDummyTransaction } from '../../../helpers/transaction_creator';
+import { ActionCreators, initialState, reducer } from '../../../reducers/transactions';
+
+import Transactions from '../../organism/Transactions';
+
+const dummyTxs = (new Array(50)).fill(0).map(createDummyTransaction);
+
+afterEach(cleanup);
+
+const Wrapper: React.FC = props => {
+  const [store, dispatch] = React.useReducer(reducer, initialState);
+
+  createTransactionCtx(store, dispatch);
+
+  if (store.transactions.length === 0) {
+    dummyTxs.map(e => dispatch(ActionCreators.addTransaction(e)));
+  }
+
+  return (
+    <TransactionCtx.Provider value={{store, dispatch}}>
+      {props.children}
+    </TransactionCtx.Provider>
+  );
+};
```

Comment:
Just FYI, I have made a GlobalWrapper in a pending PR, which can be used in stead.